numpy/testing/_private/utils.py

```diff
@@ -510,20 +511,24 @@ def assert_almost_equal(actual,desired,decimal=7,err_msg='',verbose=True):
     >>> import numpy.testing as npt
     >>> npt.assert_almost_equal(2.3333333333333, 2.33333334)
     >>> npt.assert_almost_equal(2.3333333333333, 2.33333334, decimal=10)
-    ...
-    <type 'exceptions.AssertionError'>:
-    Items are not equal:
-     ACTUAL: 2.3333333333333002
-     DESIRED: 2.3333333399999998
+    Traceback (most recent call last):
+        ...
+    AssertionError:
+    Arrays are not almost equal to 10 decimals
+     ACTUAL: 2.3333333333333
+     DESIRED: 2.2.33333334
```

Comment:
The desired value looks wrong with two decimals--probably a typo. I think you're right that refguide is now working as intended for the modules you've touched, but it will only be sensitive to the "type" of exception object, and apparently not to much beyond that.

I think @mattip recently observed this too. Maybe this is ok, but maybe our wish-list may include a bit more stringency than just checking the category of the error.

Reply:
Oops, fixed. We should definitely disable `IGNORE_EXECPTION_DETAIL` in refguide, but let's save that for another PR.